mantle/cmd/kola/kola.go

```diff
@@ -194,17 +193,15 @@ func runRun(cmd *cobra.Command, args []string) error {
 
 func runRerun(cmd *cobra.Command, args []string) error {
 	var patterns []string
-	data, err := reporters.DeserialiseReport(filepath.Join(kola.Options.CosaWorkdir, "tmp/kola/reports/report.json"))
+	file, err := os.Open(filepath.Join(kola.Options.CosaWorkdir, "tmp/kola/rerun-data"))
```

Comment:
probably should make this `rerun-data.txt` or something to indicate the file format.